django/db/models/query.py

```diff
@@ -1292,6 +1292,19 @@ def __init__(self, lookup, queryset=None, to_attr=None):
         self.queryset = queryset
         self.to_attr = to_attr
 
+    def __getstate__(self):
+        """
+        Modifies the pickling behaviour of the object.
```

Comment:
Not sure this docstring helps, doesn't it describe the purpose of all `__getstate__` methods?